lib/services/locale.dart

```diff
@@ -1,8 +1,9 @@
 part of appwrite;
 
-    /// The Locale service allows you to customize your app based on your users'
-    /// location.
+/// The Locale service allows you to customize your app based on your users&#039;
```

Comment:
Not cticital, but may result in bigger problem later. Is Was this on purpose? it as just `'` previously.